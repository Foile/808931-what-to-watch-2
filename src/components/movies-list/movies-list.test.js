import React from 'react';
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list";

it(`MoviesList correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<BrowserRouter>
    <MoviesList
      movies ={[
        {id: 1, name: `Macbeth`, previewImage: ``, previewVideoLink: ``},
        {id: 2, name: `Aviator`, previewImage: ``, previewVideoLink: ``}
      ]}
      onHeaderClick = { jest.fn() }
      onMouseEnter = { ()=> jest.fn() }
    />
  </BrowserRouter>
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
