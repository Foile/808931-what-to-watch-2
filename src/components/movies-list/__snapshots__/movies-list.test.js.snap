// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList correctly renders after relaunch 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="/films/1"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          poster=""
          src=""
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        Macbeth
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="/films/2"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <video
          height={175}
          poster=""
          src=""
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        Aviator
      </h3>
    </a>
  </article>
</div>
`;
